engine/chainstate/chainstate.go

```diff
@@ -81,12 +81,7 @@ func (c *Client) QueryTransaction(
 		return nil, spverrors.ErrInvalidRequirements
 	}
 
```

Comment:
```go
if len(id) < 50 {
		return nil, spverrors.ErrInvalidTransactionID
	} else if !c.validRequirement(requiredIn) {
		return nil, spverrors.ErrInvalidRequirements
	}
```

think you should also fix this value to something like this 

```go
if len(id) < 50 {
		return nil, spverrors.ErrInvalidTransactionID
	} else if !c.validRequirement(requiredIn) {
		return nil, spverrors.ErrInvalidRequirements
	}
```

Reply:
This is not my code and it's not relevant to this task. 
Besides - this whole package will be "zaorany" soon 🙈

Reply:
then maybe needs separate jira to correct this ???

Reply:
Apparently it is `SPV-1061`